server/src/main/scala/actors/DrtSystem.scala

```diff
@@ -421,8 +421,9 @@ case class DrtSystem(actorSystem: ActorSystem, config: Configuration, airportCon
         val lgwSasToKey = params.maybeLGWSASToKey.getOrElse(throw new Exception("Missing LGW SAS Key for To Queue"))
         val lgwServiceBusUri = params.maybeLGWServiceBusUri.getOrElse(throw new Exception("Missing LGW Service Bus Uri"))
         LGWFeed(lgwNamespace, lgwSasToKey, lgwServiceBusUri)(system).source()
-      case "BHX" if ! params.bhxIataEndPointUrl.isEmpty =>
-        BHXFeed(params.bhxIataUsername, params.bhxIataEndPointUrl)(system)
+
+      case "BHX" if !params.bhxIataEndPointUrl.isEmpty =>
+        BHXFeed(BHXClient(params.bhxIataUsername, params.bhxIataEndPointUrl), 30 seconds, 1 milliseconds)(system)
```

Comment:
I'd have thought you wouldn't need the `(system)` as it's likely available implicitly. Apologies if you've already checked this and it wasn't!

Reply:
yeah, didn't seem to work for some reason, I am not sure why.